src/graphics/context/webgl.rs

```diff
@@ -93,6 +93,28 @@ impl Texture {
     }
 }
 
+pub struct Framebuffer {
+    fb: WebGLFramebuffer,
+}
+
+impl Framebuffer {
+    pub fn new(context: &mut crate::Context, texture: &Texture) -> Option<Framebuffer> {
+        let gl_ctx = &mut context.gfx_context.webgl_context.gl_ctx;
+
+        let tex = &texture.texture;
+        let fb = gl_ctx.create_framebuffer()?;
+
+        gl_ctx.bind_texture(gl::TEXTURE_2D, Some(tex));
+        gl_ctx.bind_framebuffer(gl::FRAMEBUFFER, Some(&fb));
+        gl_ctx.framebuffer_texture2_d(gl::FRAMEBUFFER, gl::COLOR_ATTACHMENT0, gl::TEXTURE_2D, Some(tex), 0);
+        gl_ctx.bind_framebuffer(gl::FRAMEBUFFER, None);
```

Comment:
This is the setup, as far as I can tell this is following the tutorial 1 to 1. Texture is created, it's being attached to the framebuffer, all is good.